sdk/src/types/api/core/response.rs

```diff
@@ -112,6 +112,8 @@ pub struct IssuanceBlockHeaderResponse {
     pub shallow_like_parents: ShallowLikeParents,
     /// The slot index of the latest finalized slot.
     pub latest_finalized_slot: SlotIndex,
+    /// TODO: help
```

Comment:
```suggestion
    /// A Slot Commitment contains a summary of a slot.
```
from https://github.com/iotaledger/tips-draft/blob/tip46/tips/TIP-0046/tip-0046.md#slot-commitment

Reply:
Oh, that's what I was looking for. So this raises a question. In the core API, the slot commitment has a protocol version field, but in this TIP it does not. What do?

Reply:
I can't find a protocol version field in the core API

Reply:
It's in the `Commitment` definition

Reply:
Ah I was looking for `protocolVersion`, but it's just called `version`
In iota.go it's also there https://github.com/iotaledger/iota.go/blob/8290f88bd360293b56595ed03cfe2253bd5c4af9/commitment.go#L21
So I guess it's just missing in the other TIP, added a comment https://github.com/iotaledger/tips-draft/pull/20/files#r1296134085

Reply:
Okay. Regarding the comment, I was hoping for something that describes the function of this field in this struct.

Reply:
I guess it's required so the block issuer can use it to calculate the commitment id which is required in the block
But then maybe the id alone would be sufficient? Also not sure

Reply:
Got added here https://github.com/iotaledger/tips-draft/pull/20/files/39b6de8c15eb8055afde141df98f9edc3ccdf0c9..422beb7dbca565bbe0bf2369d727daa6e6fa2be7

Reply:
Yeah, still no idea what the comment should be 😆

Reply:
If it's missing in the TIP, we should ask them to add it

Reply:
There's a comment but it's not useful

Reply:
I just added something, hopefully it's true lol